### DISCUSSION TITLE
p/1cc0a366faf84858b7de600f56da7cb5/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#29] [Announcements]
# p/1cc0a366faf84858b7de600f56da7cb5/

真是疯了，吾辈发现网络上有太多的笔记工具了。 起因吾辈在大约半年前选择开始使用笔记工具，记录自己的日常知识碎片，并最终选择了 Joplin，也花费了许多时间为其编写 VSCode 插件。就在今天，吾辈在编写插件的一个功能时，需要查看类似插件是如何实现的时候，发现了许多类似的插件存在，甚至包括吾辈想在 VSCode 中管理和编辑笔记的想法也已然有人实现。 让我们看一下玩家有哪些 老牌产品  Ever

https://blog.rxliuli.com/p/1cc0a366faf84858b7de600f56da7cb5/

**Comment by windily-cloud** [comment]
嘎然而止......

**Comment by temberature** [comment]
戛然而止[Lol]

**Reply by rxliuli** [reply]
吾辈只是最终选择了 joplin+vscode

**Comment by brooke-yucx** [comment]
国内版印象笔记现在甚至不支持导出enex了，一个软件可以一直支持导出开放格式的梦想也破灭了。我已经转到Markdown+VSCode+OneDrive了，都给印象笔记整PTSD了😑

**Reply by rxliuli** [reply]
吾辈使用的是 joplin + vscode，之前做了一些 joplin 的衍生工具，现在在尝试泛化，参考：

https://mark-magic.rxliuli.com/

**Comment by SmallSnowRXS** [comment]
入坑 joplin ！
